draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4303,26 +4307,36 @@ better still.
 
 <section title="Changes from RFC 5245">
 
-<t> Following is the list of changes from RFC 5245
-<list style="symbols">
-<t> The specification was generalized to be more usable with any
-protocol and the parts that are specific to SIP and SDP were moved to
-a SIP/SDP usage document <xref
-target="I-D.ietf-mmusic-ice-sip-sdp"/>. </t>
-<t> Default candidates, multiple components, ICE mismatch detection,
-subsequent offer/answer, and role conflict resolution were made
-optional since they are not needed with every protocol using ICE. </t>
-<t>
-With IPv6, the precedence rules of RFC 6724 are used instead of the
-obsoleted RFC 3483 and using address preferences provided by the host
-operating system is recommended. </t>
-<t> Candidate gathering rules regarding loopback addresses and IPv6
-addresses were clarified. </t>
-
+<t> The purpose of this specification is to:
+<list>
```

Comment:
should this also be with style="symbols"

Reply:
Yes.